paper/src/main/java/rocks/gravili/notquests/paper/events/hooks/CitizensEvents.java

```diff
@@ -209,10 +217,9 @@ private void onNPCClickEvent(NPCRightClickEvent event) { //Disconnect objectives
                     if (npcToEscort != null) {
                         if (npcToEscort.isSpawned() && (npcToEscort.getEntity().getLocation().distance(player.getLocation()) < 6)) {
                             activeObjective.addProgress(1, nqNPC);
-                            final String mmNpcName = main.getMiniMessage().serialize(LegacyComponentSerializer.legacyAmpersand().deserialize(npcToEscort.getName()));
 
                             player.sendMessage(main.parse(
-                                "<GREEN>You have successfully delivered the NPC <highlight>" + mmNpcName
+                                    main.getLanguageManager().getString("chat.npc.delivery-of-npc-complete", questPlayer)
```

Comment:
you forgot to add the map to replace the NPC name there

Reply:
ah yes thank you will ammend